lib/tools/markusapi.py

```diff
@@ -120,24 +120,25 @@ def upload_feedback_file(self, assignment_id, group_id, title, contents):
         path = self.get_path(assignment_id, group_id) + 'feedback_files'
         return self.submit_request(params, path, 'POST')
 
-    def upload_test_script_result(self, assignment_id, group_id, results):
-        """ (Markus, int, str, dict) -> list of str """
```

Comment:
@adisandro I don't think you realize that the part after the `->` is the return type of the funtion, so this shouldn't be changed.

Also you should use the type `list` not `array`.